### PR TITLE
Make coming soon section 4-col

### PR DESCRIPTION
**Pull request by trishalim**

### CODE REVIEW

File: homepage/homepage/components/home/ComingSoonSection.tsx [hunk 1]
```diff
@@ -15,39 +15,45 @@ export function ComingSoonSection() {
     <div>
       <SectionHeader title="More features coming soon" />
 
-      <GappedGrid>
-        <GridCard>
-          <H3>Cursors & carets</H3>
-          <P className="text-lg">Ready-made spatial presence.</P>
+      <GappedGrid cols={4}>
+        <Card className="p-4">
+          <h3 className="text-stone-900 font-medium md:text-base dark:text-stone-100 mb-2">
```
Comment (bensleveritt) [inline homepage/homepage/components/home/ComingSoonSection.tsx]: Should we capture this heading style into a component? If only to keep track of what styles we're using

**Review by bensleveritt — APPROVED** [review]
LGTM

File: homepage/homepage/components/home/ComingSoonSection.tsx (hunk 1 above)
Comment (trishalim) [reply homepage/homepage/components/home/ComingSoonSection.tsx]: agreed

Comment (trishalim) [reply homepage/homepage/components/home/ComingSoonSection.tsx]: I placed in inside a component `FeatureCard`. I don't want to change our `H3` component. Styling based on headings works for article content, but feels very limiting on a landing page.

**Pull request merged**